docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -90,13 +90,14 @@ For a list of topic properties that you can update using the CLI, see the `rhoas
 * You have created a Kafka instance with at least one Kafka topic in {product-kafka}.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topics that you want to configure.
-. In the *Topics* page, click the name of the topic that you want to review or edit.
-+
-Alternatively, select the options icon (three vertical dots) for the relevant topic and click *Edit*.
-. Click *Edit properties*, review the current topic properties, and adjust any editable topic properties as needed.
+. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances*.
+. Click the Kafka instance *Name* that contains the topic you want to configure.
+. Select the *Topics* page.
+. Select the options icon (three vertical dots) for the relevant topic *Name* and click *Edit* to review the current topic properties, and adjust any editable topic properties as needed.
 . Click *Save* to finish.
 
+NOTE: You can also edit a specific topic from your Kafka instance. Select the *Topics* page and click the name of the Kafka instance that contains the topic that you want to configure. Select the *Properties* page, and click *Edit properties*. Review the current topic properties, adjust any editable topic properties and click *Save*.
```

Comment:
This isn't right. You click a topic on the *Topics* page, not a Kafka instance.

In any case, I think something shorter is sufficient. Something like:

NOTE: You can also edit topic properties by selecting the *Properties* tab within the topic.